websocket/deviceShadowUpdateChecker.ts

```diff
@@ -100,7 +100,7 @@ export const createDeviceUpdateChecker = async (
 		const defaultStep = config['default'] ?? [
 			{ count: Number.MAX_SAFE_INTEGER, interval: 5 },
 		]
-		const deviceModelHash = hashSHA1(device.model)
+		const deviceModelHash = hashSHA1(device.model ?? '')
```

Comment:
```suggestion
		const deviceModelHash = hashSHA1(device.model)
```

This should never happen. If a device has no model it is badly configured, do not fetch shadow for these devices.

Reply:
I do agree that it should not happen, but I found this case during the load test. This happened before I add `attribute_exists(#connectionId)` into the condition expression per this https://github.com/hello-nrfcloud/backend/blob/c1bfc89e39a270e86f7ce708d301f85b62453a03/websocket/connectionsRepository.ts#L143.

Without having the connection Id in condition expression, it just adds data as a new item without model. 

Even though I have fixed this, but just in case the model does not exist, it will throw the error because hash function requires string, not undefined.

Reply:
But then it can be reverted now, because this will no longer happen.